UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.IntegrationTests/RadioNavigationalWarning/BaseRNWTest.cs

```diff
@@ -0,0 +1,172 @@
+﻿using FakeItEasy;
+using Microsoft.AspNetCore.Http;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using UKHO.MaritimeSafetyInformation.Common;
+using UKHO.MaritimeSafetyInformation.Common.Configuration;
+using UKHO.MaritimeSafetyInformation.Web.Services;
+using radioNavigationalWarningDto = UKHO.MaritimeSafetyInformation.Common.Models.RadioNavigationalWarning.DTO;
+
+namespace UKHO.MaritimeSafetyInformation.IntegrationTests.RadioNavigationalWarning
+{
+    public class BaseRNWTest
+    {
+        public readonly RadioNavigationalWarningsContext _fakeContext;
+        public IOptions<RadioNavigationalWarningConfiguration> _fakeRadioNavigationalWarningConfiguration;
+        public IHttpContextAccessor _fakeHttpContextAccessor;
+        public ILogger<RNWService> _fakeLoggerRnwService;
```

Comment:
Should be `protected`, `readonly` and Pascal Case because they're not private:
```suggestion
        protected readonly RadioNavigationalWarningsContext FakeContext;
        protected readonly IOptions<RadioNavigationalWarningConfiguration> FakeRadioNavigationalWarningConfiguration;
        protected readonly IHttpContextAccessor FakeHttpContextAccessor;
        protected readonly ILogger<RNWService> FakeLoggerRnwService;
```

All the methods in this bas class can be `protected` too.